README.md

```diff
@@ -1,44 +1,60 @@
-# JIRA CLI [![Gem Version](https://badge.fury.io/rb/jira-cli.png)](http://badge.fury.io/rb/jira-cli)
+# JIRA CLI [![Gem Version](https://badge.fury.io/rb/jira-cli2.png)](http://badge.fury.io/rb/jira-cli2)
 
 Ruby gem CLI tool used to manage JIRA workflows leveraging git
 
 * * *
 
-[![Circle CI](https://circleci.com/gh/drn/jira-cli.svg?style=svg)](https://circleci.com/gh/drn/jira-cli)
-[![Dependency Status](https://gemnasium.com/darrenli/jira-cli.png)](https://gemnasium.com/darrenli/jira-cli)
-[![Code Climate](https://codeclimate.com/github/darrenli/jira-cli.png)](https://codeclimate.com/github/darrenli/jira-cli)
+[![Dependency Status](https://gemnasium.com/SDarayan/jira-cli.png)](https://gemnasium.com/SDarayan/jira-cli)
 
 ### Available Commands
 
     jira all             # Describes all local branches that match JIRA ticketing syntax
+    jira assign          # Assign a ticket to a user
     jira attachments     # View ticket attachments
-    jira browse          # Opens the current input ticket in your browser
```

Comment:
There's no reason to leave them commented especially since this needs to be written to be cross-platform. If you don't remove them, I will